docs/basics/sso/setting-up-sso-special-cases.md

```diff
@@ -0,0 +1,37 @@
+---
+id: setting-up-sso-special-cases
+title: Setting Up SSO - Special Cases
+sidebar_label: Setting Up SSO - Special Cases
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<p><span className="sauceDBlue">Enterprise Plans only</span></p>
+
+## Single Identity Provider and Multiple Organizations at Sauce Labs
+
+Some identity providers do not allow creating more than one SAML integration with the same service provider. In other words, they **require unique entity ID of service providers**.
+
```

Comment:
:::note
 Integrating more than two  Sauce Labs organizations with the same  Identity Provider is not supported. If you have more than two organizations that you need integration with the same  Identity Provider, contact your Customer Success Manager or [Sauce Labs Support](mailto:help@saucelabs.com). 
 :::

Reply:
I would move the note to the top of the page.

Reply:
fixed